specs/vendor_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative './spec_helper'
+require_relative '../far_mar.rb'
```

Comment:
If you move this line to `spec_helper`, you won't need to do it in every spec file.